bee-tangle/src/vec_set.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#![allow(dead_code)]
```

Comment:
What's not being used and why not to remove it ?

Reply:
`new` is unused. I think a `new` method is pretty standard for a data structure like this though. I can remove it for now though given that the rest of the API isn't really fleshed out.